relay-event-schema/src/protocol/span.rs

```diff
@@ -67,6 +67,9 @@ pub struct Span {
     #[metastructure(pii = "true")]
     pub data: Annotated<Object<Value>>,
 
+    /// Tags generated by Relay. These tags are a superset of the tags set on span metrics.
+    pub sentry_tags: Annotated<Object<String>>,
```

Comment:
Should we run PII here?

Reply:
We have seen cases where the `group` tag got replaced by `[Filtered]` because of PII scrubbing. So no: We want these tags to contain the same values as the span metrics, and exclude them from PII scrubbing deliberately.